### PR TITLE
gpxsee: Migrate to QT6

### PR DESCRIPTION
**Pull request by nazar-stasiv**
**Summary**
- replace QT5 with QT6

**Test Plan**
- open map file, zoom in and out

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/g/gpxsee/package.yml [hunk 1]
```diff
@@ -10,12 +10,12 @@ summary    : a Qt-based GPS log file viewer and analyzer that supports all commo
...
-    - pkgconfig(Qt5Svg)
-    - pkgconfig(Qt5UiTools)
+    - pkgconfig(Qt6Positioning)
+    - pkgconfig(Qt6SerialPort)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6UiTools)
 setup      : |
-    %qmake CONFIG+=release gpxsee.pro PREFIX=%PREFIX%
+    qmake6 CONFIG+=release gpxsee.pro PREFIX=%PREFIX%
 build      : |
```
Comment (malfisya) [inline packages/g/gpxsee/package.yml:19]: Can you try build it with :
```
clang      : yes
optimize   :
    - speed
    - thin-lto
 ```
To make it in line with others qt6 packages

**Review by malfisya — DISMISSED** [review]
Please also add [monitoring.yml](https://help.getsol.us/docs/packaging/monitoring.yml). We want to actively use it going forward.

File: packages/g/gpxsee/package.yml (hunk 1 above)
Comment (nazar-stasiv) [reply packages/g/gpxsee/package.yml:19]: Sounds reasonable. 

Want to do something useful too? Evaluate how current version of the package performs on bigger maps, like continent or country[1].  Later I'll update PR for you to evaluate if new upgrades and optimisations have desired effect. 

1. I don't have broadband for large downloads, but pre-rendered maps available [here](https://garmin.bbbike.org) 

Thanks

**Comment by nazar-stasiv** [comment]
> Please also add [monitoring.yml](https://help.getsol.us/docs/packaging/monitoring.yml). We want to actively use it going forward.

all right, I'll include that file with the next version upgrade patch.

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**